### ISSUE TITLE
Misc issues relating to Encounters

### ISSUE THREAD

**Issue by CarlosNZ** [openmsupply/open-msupply#1158]
About to make a PR for #676, but it's getting a bit all-over-the-place so I'll make a list here of remaining tweaks and other little things I've discovered along the way:

- [x] ~~Encounters list has two different "Status" columns -- presumably there should just be the one (Which one?)~~ They both have a purpose, maybe one should be renamed?
- [x] Encounters list has both "Type" and "Program" columns, should probably be the same? Clemens: its the encounter type and the program the encounter belongs to. However, the program column should display the program name from document registry to make it look nicer. (Rename "Type" to "Encounter" as per Telegram discussion) (LOW PRIORITY)
- [x] Clarify the exact fields required for the Patient data in the Header (and which if any should be editable)
- [x] Add separate "Entered by" field (once "User" node added to Encounter GraphQL endpoint)
- [x] Add separate "Created time" field (once #1156 is merged)
- [x] Figure out how to handle Encounter status -- where it is displayed, when it is entered, and if it can be customised with JSON Forms #1283 
- [x] Clarify exact fields required for Side Panel display (and which, if any, are editable)

**Comment by DhanyaHerath** [comment]
Triage: @clemens-msupply to check whether some of this is already in other issues

**Comment by CarlosNZ** [comment]
Okay, this is getting a bit complicated for me, so let's have a proper discussion about them next week when @richardmoizeau is in the office?

**Comment by CarlosNZ** [comment]
From IRL discussions:

> Encounters list has both "Type" and "Program" columns, should probably be the same? Clemens: its the encounter type and the program the encounter belongs to. However, the program column should display the program name from document registry to make it look nicer. (Rename "Type" to "Encounter" as per Telegram discussion) (LOW PRIORITY)

For now, rename to "Additional Information", and show the first item of the "events" field (of "Encounter Status" type)

>  Clarify the exact fields required for the Patient data in the Header (and which if any should be editable)

Leave as is

>  Add separate "Entered by" field (once "User" node added to Encounter GraphQL endpoint)

Add to schema

> Clarify exact fields required for Side Panel display (and which, if any, are editable)

- Entered by
- Entered on (created on)
- Don't show things duplicated in Headers (e.g. Clinician)
- Previous encounters should open in new tab

**Comment by CarlosNZ** [comment]
> * Figure out how to handle Encounter status -- where it is displayed, when it is entered, and if it can be customised with JSON Forms

> * Encounters list has both "Type" and "Program" columns, should probably be the same? Clemens: its the encounter type and the program the encounter belongs to. However, the program column should display the program name from document registry to make it look nicer.

@roxy-dao I think you may have addressed these two as part of other issues. Can you confirm? (Or see any other points that you've dealt with?)

**Comment by roxy-dao** [comment]
> 

I think it's just those two~

**Comment by CarlosNZ** [comment]
Closed by #1483

**Comment by DhanyaHerath** [comment]
I can't believe it's closed 😮  What a day! 🎉 🎉 🎉
